lib/ansible/modules/cloud/docker/docker_compose.py

```diff
@@ -94,47 +96,47 @@
       - smart
   build:
     description:
-      - Use with state I(present) to always build images prior to starting the application.
-      - Same as running docker-compose build with the pull option.
+      - Use with C(state) I(present) to always build images prior to starting the application.
+      - Same as running I(docker-compose build) with the pull option.
       - Images will only be rebuilt if Docker detects a change in the Dockerfile or build directory contents.
       - Use the C(nocache) option to ignore the image cache when performing the build.
       - If an existing image is replaced, services using the image will be recreated unless C(recreate) is I(never).
     type: bool
     default: no
   pull:
     description:
-      - Use with state I(present) to always pull images prior to starting the application.
-      - Same as running docker-compose pull.
+      - Use with C(state) I(present) to always pull images prior to starting the application.
+      - Same as running I(docker-compose pull).
       - When a new image is pulled, services using the image will be recreated unless C(recreate) is I(never).
     type: bool
     default: no
     version_added: "2.2"
   nocache:
     description:
-      - Use with the build option to ignore the cache during the image build process.
+      - Use with the C(build) option to ignore the cache during the image build process.
     type: bool
     default: no
     version_added: "2.2"
   remove_images:
     description:
-      - Use with state I(absent) to remove the all images or only local images.
+      - Use with C(state) I(absent) to remove the all images or only local images.
     type: str
     choices:
         - 'all'
         - 'local'
   remove_volumes:
     description:
-      - Use with state I(absent) to remove data volumes.
+      - Use with C(state) I(absent) to remove data volumes.
     type: bool
     default: no
   stopped:
     description:
-      - Use with state I(present) to leave the containers in an exited or non-running state.
+      - Use with C(state) I(present) to stop containers.
```

Comment:
```suggestion
      - Use with C(state) I(present) to stop all containers.
```
similarly to `restarted`.